shared/constants/platform.desktop.js

```diff
@@ -123,24 +123,31 @@ function findCacheRoot(): string {
 }
 
 function logDir(): string {
+  // See LogDir() functions in go/libkb/home.go.
+  //
+  // TODO: darwin and win32 cases are inconsistent with their LogDir()
+  // counterparts. Fix this.
   switch (process.platform) {
     case 'darwin':
       return `${getenv('HOME', '')}/Library/Logs`
     case 'linux':
-      const linuxDefaultRoot = `${getenv('HOME', '')}/.local/share`
-      return `${getenv('XDG_DATA_HOME', linuxDefaultRoot)}/${envedPathLinux[runMode]}`
+      return findCacheRoot()
     case 'win32':
       return `${getenv('LOCALAPPDATA', '')}\\${envedPathWin32[runMode]}`
   }
   throw new Error(`Unknown platform ${process.platform}`)
 }
 
 function logFileName(): string {
+  // See DesktopLogFileName in go/libkb/constants.go.
+  //
+  // TODO: darwin and win32 cases are inconsistent with
```

Comment:
darwin just happens to work for prod mode. not sure why win32 isn't bitten by the difference in case. @zanderz do you know?